swift/Rakefile

```diff
@@ -140,6 +141,8 @@ def create_uikit_targets(project, demo, aggregate_target)
 
     target_add_carthage_framework(target, :ios, "Ice.framework", true)
     target_add_carthage_framework(target, :ios, "PromiseKit.framework", true)
+    target_add_carthage_framework(target, :ios, "MessageKit.framework", true)
+    target_add_carthage_framework(target, :ios, "InputBarAccessoryView.framework", true)
 
```

Comment:
You should move this to `$demo_extra_frameworks` and only add them to `iOS/chat`

```
"iOS/chat" => ["Glacier2.framework", 
                "MessageKit.framework", 
                "InputBarAccessoryView.framework"],
```